### PR TITLE
Done some changes in navbar

### PR DESCRIPTION
**Pull request by shubham-tyagi**
Signed-off-by: shubham-tyagi <raj.tyagi2000@gmail.com>

**Description**
I have tried and did some minor tweaks in navbar :
- removed the link from the nav bar subitem for the page on which page we are.
- highlighted the link on whose sublink page we are.
- opening of the docs in a different tab, added the icon in front of docs.
- added cursor as a pointer for navbar link (we had different cursor for community and resources previously) 

**Notes for Reviewers**
If any tweak doesn't suit the navbar please let me know I will remove that tweak.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by warunicorn19** [comment]
@shubham-tyagi let's discuss these changes on the websites' call tomorrow.

**Comment by shubham-tyagi** [comment]
sure @warunicorn19

File: _sass/navigation.scss [hunk 1]
```diff
@@ -41,9 +42,12 @@ a.nav-text:hover {
 a.subnav-item:hover {
 	color: rgba(0, 179, 159, 1) !important;
 }
-
+a.subnav-item.active{
```
Comment (shubham-tyagi) [inline _sass/navigation.scss:45]: I will be removing this before the code gets merged, this selector is of no use.

**Comment by warunicorn19** [comment]
@shubham-tyagi could you please make the changes as discussed in the websites' call

**Comment by shubham-tyagi** [comment]
Sure @warunicorn19

**Comment by leecalcote** [comment]
Thanks for working on this, @shubham-tyagi

**Comment by shubham-tyagi** [comment]
@warunicorn19 please do have a look and let me know if any other change is required.

**Review by warunicorn19 — APPROVED** [review]
LGTM! Thanks @shubham-tyagi

**Pull request merged**